pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -4381,4 +4381,13 @@ If you choose Disable, you can enable Auto-select later with the "Refine &gt; Ad
   <data name="SummaryRelativeAbundanceGraphPane_UpdateAxes_Molecule_Rank" xml:space="preserve">
     <value>Molecule Rank</value>
   </data>
+  <data name="ValueInvalidModException_ValueInvalidModException_Unable_to_add_peptide_modification___0_____1_" xml:space="preserve">

```

Comment:
All the other CommandArgs.ValueInvalid exception messages are in Resources, so I put it there for consistency. It does seem like they ALL should move to CommandArgUsage.resx. I suppose they haven't moved because they're used somewhere else in Skyline, many of them in tests, I think. @nickshulman does the resx split logic ignore the tests when deciding where to move the strings?

Reply:
If a string was being used by source files in more than one folder, or was also being used by source files in a different project, then the string was left in Resources.resx instead of being moved to a different .resx file.

Reply:
Should we modify that logic to ignore tests? Since those are always going to be in a different project, and often need to use the string resources so the tests are localization aware. But using the strings from the tests doesn't seem like a good reason not to move them to a more localized resx.

Reply:
The reason that strings did not get moved when the string was being used by the test file is that my script would have needed to modify the test file and add a "using" statement.
I did not want to implement the logic for adding a "using" statement.
(After my tool ran, there were many source files that needed to have the "using pwiz.Skyline.Properties;" line removed, and I did that by hand)